WalletWasabi/WalletWasabi.csproj

```diff
@@ -27,6 +27,9 @@
   </ItemGroup>
 
   <ItemGroup>
+    <None Update="Hwi\Software\digests.txt">
```

Comment:
Are these leftover?

Reply:
No, why? The `digests.txt` files contain the hashes for linux, windows and osx.

Reply:
We made a decision that you hash the existing binaries we have. But you did this. And this also has to be manually updated every time you update the software. This will be forgotten and we will scratch our heads that why the heck does it not update the software. That's not good.

Reply:
You said the hashes will be different from OS to OS, so if I want to ship a new version of the software I have to do the hashing on different operating systems? Also how do you know every Linux distro are hashing the same way as the other?  

This is why we made a decision that you hash the existing software and compare against that, but you did not do that. Why?

Reply:
>  if I want to ship a new version of the software I have to do the hashing on different operating systems. 

No. You can hash all the three files in the same OS.

>how do you know every Linux distro are hashing the same way as the other?

Because a hash is the content fingerprint and that only changes if the content changes (it is not OS dependent)

> Hashes must be generated on the users' computers, because they differ, no?

No. It doesn't depend on the computer.

> This is why we made a decision that you hash the existing software and compare against that, but you did not do that. Why?

I am not sure I am following you here. What do you mean by "hash the existing software and compare against that"? Isn't that what I did? 

I understand that creating the `digests.txt` files manually is a pain because we need to go and hash the files but we can also automate the procedure in the Packager so, the hashes can be computed, and the digests.txt files created, on release time.

Reply:
Quoting you from slack: "the hash doesn't work because it depends on the OS"

Reply:
I added digest creation with packager. It's the same on my Windows so it should work fine.

Reply:
https://github.com/zkSNACKs/WalletWasabi/commit/b4dfa39fe00e2b9fa1bf5777424ca1e48f229c30